src/Executable.js

```diff
@@ -105,6 +111,63 @@ export default class Executable {
         return this;
     }
 
+    /**
+     * @param {number} minBackoff
+     */
+    setMinBackoff(minBackoff) {
+        if (this._maxBackoff != null && minBackoff >= this._maxBackoff) {
```

Comment:
I believe this should be `minBackoff > this._maxBackoff` since those two fields being equal would indicate backoff is linear instead of exponential.